debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -487,6 +487,36 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
                     + " matched against this regular expression. If matched the error is changed to retriable.")
             .withDefault(false);
 
+    public static final Field CONNECT_ERROR_MAX_RETRIES = Field.create("connect.error.max.retries")
+            .withDisplayName("Max retries on connection errors")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 998))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDefault(0)
+            .withValidation(Field::isInteger)
+            .withDescription("The maximum number of times to retry on connection errors before failing the task (-1 - no limit). Defaults to 0 (no retries).");
+
+    public static final Field CONNECT_ERROR_INITIAL_RETRY_DELAY = Field.create("connect.error.retry.initial.delay.ms")
+            .withDisplayName("Initial retry delay when encountering connection errros (in ms)")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 997))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDefault(300)
+            .withValidation(Field::isPositiveInteger)
+            .withDescription("This value will be doubled upon every retry but won't exceed connect.error.max.retry.delay.ms");
+
+    public static final Field CONNECT_ERROR_MAX_RETRY_DELAY = Field.create("connect.error.max.retry.delay.ms")
+            .withDisplayName("Max delay when encountering connection errros (in ms)")
```

Comment:
```suggestion
            .withDisplayName("Max delay when encountering connection errors (in ms)")
```

Reply:
The display name here seems to be almost the same length as the description text below. The display name is used for the Debezium UI to label the field. Can we use a shorter, but still descriptive name for the field?
Furthermore, are we capturing Kafka Connect options like this? If not, aren't we re-using a reserved Kafka Connect property (which implies a typo maybe)? Kafka Connect has `errors.retry.delay.max.ms` for connectors (plural!).

Reply:
Yes, those names resemble those of Kafka Connect. But it is on purpose, as they should only be used with the embedded engine and/or Debezium Server.